src/fgimic/jfield.f90

```diff
@@ -627,12 +635,29 @@ subroutine get_property(this)
         close(15)
         close(16)
 
+        ! get cell information for printing vtu file
+        ! read element file: first number in first line is number of lines
+        inquire(FILE='grid.1.ele', EXIST=elements_exists)
+        if(elements_exists) then
+          open(GRIDELE, file='grid.1.ele')
+          ! read(GRIDELE, '(3i4)') ncells, dummy, dummy
+          read(GRIDELE, *) ncells, dummy, dummy
+          ! write(*,*) "ncells ", ncells
+          allocate(cells(4,ncells))
+          do i=1,ncells
+            ! read(GRIDELE,'(5i7)') dummy, cells(1:4, i)
```

Comment:
i'd probably delete a few outcommented lines here and further down

Reply:
Ok. When everything works we can do a clean up.